conf/modules.config

```diff
@@ -871,6 +871,31 @@ process {
         ext.prefix = { "${meta.assembler}-MetaBinner-${meta.id}" }
     }
 
+    withName: SEMIBIN_SINGLEEASYBIN {
+        publishDir = [
+            [
+                path: { "${params.outdir}/GenomeBinning/SemiBin2/" },
+                mode: params.publish_dir_mode,
+                pattern: 'bins/*.fa.gz',
+            ],
+            [
+                path: { "${params.outdir}/GenomeBinning/SemiBin2/log" },
+                mode: params.publish_dir_mode,
+                pattern: '*.log',
+            ],
+        ]
+        ext.prefix = { "${meta.assembler}-SemiBin2-${meta.id}" }
+        ext.args2  = {[
+                params.semibin_rng_seed == 0 ? "" : "--random-seed ${params.semibin_rng_seed}",
+                "--min-len ${params.min_contig_size}",
+                "--compression gz",
+                "--engine cpu",
```

Comment:
Could we add a param for this?  Seems to be a common thing to change of you have GPUs available.